docs/file-name-matches-element.md

```diff
@@ -0,0 +1,74 @@
+# Enforce that files which export a custom element are named similarly to the custom element that they export (file-name-matches-element)
+
+For code organisation, it is useful if a file name matches the contents of a class inside it. In this instance, having the file name match the name of the element.
+
+## Rule Details
+
+This rule enforces the name of the file an Element exists in matches the name of the Element itself.
+
+👎 Examples of **incorrect** code for this rule:
+
+```js
+// element.ts
+class FooBarElement extends HTMLElement {}
+```
+
+👍 Examples of **correct** code for this rule:
+
+```js
+// foo-bar-element.ts
+class FooBarElement extends HTMLElement {}
+```
+
+### Options
+
+- `transform` can define how a class name would be transformed into a file name. You can specify multiple transforms to allow multiple options. The default is `['kebab', 'pascal']`. The options are:
```

Comment:
i actually split up what was `pascal` into `pascal` and `camel` separately, making the new default `['kebab', 'camel']`

before, PascalCase was matching camelCase

Reply:
I've added `camel` and changed the default :+1: